experts-migration-commands.php

```diff
@@ -0,0 +1,742 @@
+		<p>Tel: 250.317.0293</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:patty.wellborn@ubc.ca">patty.wellborn@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+		$lou_corpuz_bosshart['title'] = 'Lou Corpuz-Bosshart Media Relations';
+		$lou_corpuz_bosshart['content'] = '<!-- wp:heading {"level":3} -->
+		<h3>Media Contact</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:paragraph -->
+		<p>Lou Corpuz-Bosshart</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Media Relations Specialist</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Tel: 604.822.2048</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Cell: 604.999.0473</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:lou.bosshart@ubc.ca">lou.bosshart@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+		$alex_walls['title'] = 'Alex Walls Media Relations';
+		$alex_walls['content'] = '<!-- wp:heading {"level":3} -->
+		<h3>Media Contact</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:paragraph -->
+		<p>Alex Walls</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Media Relations Specialist</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Tel: 604.822.4636</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Cell: 778.984.6173</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:alex.walls@ubc.ca">alex.walls@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+		$collins_maina['title'] = 'Collins Maina Media Relations';	
+		$collins_maina['content'] = '<!-- wp:heading {"level":3} -->
+		<h3>Media Contact</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:paragraph -->
+		<p>Collins Maina</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Assistant Media Relations Specialist</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Tel:  604.802.0779</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:collins.maina@ubc.ca">collins.maina@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+		$erik_rolfsen['title'] = 'Erik Rolfsen Media Relations';
+		$erik_rolfsen['content'] = '<!-- wp:heading {"level":3} -->
+		<h3>Media Contact</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:paragraph -->
+		<p>Erik Rolfsen</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Media Relations Specialist</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Tel: 604.822.2644</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Cell: 604.209.3048</p>
+		<!-- /wp:paragraph -->
+		
+		<!-- wp:paragraph -->
+		<p>Email: <a href="mailto:erik.rolfsen@ubc.ca">erik.rolfsen@ubc.ca</a></p>
+		<!-- /wp:paragraph -->';
+
+
+		$collins_post = array(
+			'post_title'	=> $collins_maina['title'],
+			'post_content'  => $collins_maina['content'],
+			'post_status'   => 'publish',
+			'post_type'		=> 'wp_block'
+
+		);
+		$collins_maina['id'] = wp_insert_post( $collins_post, false );
```

Comment:
Honestly, I know the data is literally set directly above, hardcoded, and I know this is definitely a "safe" state, but I'd really like to encourage the use of sanitization functions whenever interacting with the database. In this case, it's also possible using the appropriate functions will also help save you from any issues with weird ascii characters etc.

i.e.

```php
$collins_post = array(
	'post_title'	=> wp_strip_all_tags( sanitize_title_with_dashes( $collins_maina['title'] ) ),
	'post_content' => wp_kses_post( $collins_maina['content'] ),
	'post_status'   => 'publish',
	'post_type'	=> 'wp_block'
);
```